inc/auth.class.php

```diff
@@ -493,11 +493,11 @@ function getAlternateAuthSystemsUserLogin($authtype = 0) {
                if (count($data) === 2) {
                   list ($cookie_id, $cookie_token) = $data;
 
-                  $token = User::getToken($cookie_id, 'personal_token');
+                  $user = new User();
+                  $user->getFromDB($cookie_id);
+                  $token = $user->getAuthToken('personal_token');
```

Comment:
`personal_token` value is default one and can be removed. Same remark applies to other calls to `getAuthToken()` method.